examples/workflow_example_cep.jl

```diff
@@ -20,30 +20,30 @@ ts_seg_data = run_clust(ts_input_data;method="kmeans",representation="centroid",
 optimizer=Clp.Optimizer
 
 # tweak the CO2 level
-co2_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;descriptor="co2",co2_limit=50) #generally values between 1250 and 10 are interesting
+co2_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;co2_limit=50) #generally values between 1250 and 10 are interesting
 
 # Include a Slack-Variable
-slack_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;descriptor="slack",lost_el_load_cost=1e6, lost_CO2_emission_cost=700)
+slack_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;lost_el_load_cost=1e6, lost_CO2_emission_cost=700)
 
 
 # Include existing infrastructure at no COST
-ex_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;descriptor="ex",existing_infrastructure=true)
+ex_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;existing_infrastructure=true)
 
 # Intraday storage (just within each period, same storage level at beginning and end)
-simplestor_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;descriptor="simple storage",storage="simple")
+simplestor_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;storage="simple")
 
 # Interday storage (within each period & between the periods)
-seasonalstor_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;descriptor="seasonal storage",storage="seasonal")
+seasonalstor_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;storage="seasonal")
 
 # Transmission
-transmission_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;descriptor="transmission",transmission=true)
+transmission_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;transmission=true)
 
 # Segmentation
-seg_result = run_opt(ts_seg_data.best_results,cep_data,optimizer;descriptor="segmentation")
+seg_result = run_opt(ts_seg_data.best_results,cep_data,optimizer)
 
 # Desing with clusered data and operation with ts_full_data
 # First solve the clustered case
-design_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;descriptor="design&operation", co2_limit=50)
+design_result = run_opt(ts_clust_data.best_results,cep_data,optimizer;co2_limit=50)
```

Comment:
Just to check if I understand this: Basically how I see it is that the descriptor moved into the variable description? Great, that makes a lot of sense to me.

Reply:
yes exactly. First users were getting a little confused, what the descriptor does...